js/jsmind.js

```diff
@@ -2096,13 +2104,41 @@
                 parent_node.appendChild(d_e);
                 view_data.expander = d_e;
             }
-            if(this.opts.support_html){
-                $h(d,node.topic);
-            }else{
-                $t(d,node.topic);
+            if (!!node.topic) {
+                if(this.opts.support_html){
+                    $h(d,node.topic);
+                }else{
+                    $t(d,node.topic);
+                }
             }
             d.setAttribute('nodeid',node.id);
             d.style.visibility='hidden';
+            if (!!node.image) {
+                var imageSize = this.jm.options.image_node_size;
+                // if image is a data url, scale to imageSize
+                if (node.image.startsWith('data')) {
+                    var img = new Image();
+                    img.onload = function() {
+                        var c = document.createElement('canvas');
+                        c.width = imageSize;
+                        c.height = imageSize;
+                        var img = this;
+                        if(c.getContext) {
+                            var ctx = c.getContext('2d');
+                            ctx.drawImage(img, 2, 2, imageSize, imageSize);
```

Comment:
The user can change the default image in the options, or we can add logic to enlarge the image later. In the last case, we will need preserve the original image if we don't want get pixelated images. In my use case, I am focused in simple use for kids.